python/lsst/ts/ess/common/sensor/gill_windsonic.py

```diff
@@ -20,14 +20,15 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 __all__ = [
-    "WindSensor",
+    "WindsonicSensor",
     "DEFAULT_DIRECTION_VAL",
     "DEFAULT_SPEED_VAL",
     "GOOD_STATUS",
     "END_CHARACTER",
     "START_CHARACTER",
     "UNIT_IDENTIFIER",
     "WINDSPEED_UNIT",
+    "compute_checksum",
```

Comment:
Please consider changing these constants (DEFAULT_DIRECTION_VAL, etc.) to class variables on WindSensor (and so lowercase). That gives a nice namespace and keeps related data more closely together.

In other words, to define:
```
class WindSensor(...):
   default_direction_str = "999"
   ...
   def __init__(...)
```

Users can then access them via the class name (or via an instance, if you have one handy):
```
from sensors import WindsonicSensor

foo = WindsonicSensor.default_direction_str
# or
foo = my_sensor.default_direction_str
```

Reply:
I like the suggestion very much. As a matter of fact, I have been struggling with this since other sensor classes define constants as well and the way it is done now indeed pollutes the namespace. I have done this for this and all other sensor classes.

Reply:
Ah wait, the other sensor classes don't have constants that are exported so only the Gill Windsonic code needed to be updated.